### PR TITLE
More schemas for basic recipes

### PR DESCRIPTION
**Pull request by umairidris**
#74

### CODE REVIEW

File: templates/tfengine/recipes/org/audit.hcl [hunk 1]
```diff
@@ -12,6 +12,48 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+schema = {
```
Comment (MartinPetkov) [inline templates/tfengine/recipes/org/audit.hcl:15]: Seeing it now, I feel that having this schema here like this really helps!

File: templates/tfengine/recipes/org/audit.hcl [hunk 2]
```diff
@@ -12,6 +12,48 @@
...
+      type        = "string"
+    }
+    bucket_name = {
+      description = "Name of GCS Bucket to store 7 year audit logs."
+      type        = "string"
+    }
+    auditors = {
+      description = <<EOF
+        This group will be granted viewer access to the audit log dataset and
+        bucket as well as security reviewer permission on the entire org.
```
Comment (xingao267) [inline templates/tfengine/recipes/org/audit.hcl:41]: on the entire org --> on the root resource specified

File: templates/tfengine/recipes/org/monitor.hcl [hunk 3]
```diff
@@ -12,6 +12,36 @@
...
+    project_id = {
+      description = "ID of the project to host monitoring resources."
+      type        = "string"
+    }
+    domain = {
+      description = "Domain for the Forseti instance."
+      type        = "string"
+    }
+    cloud_sql_region = {
+      description = "Default location for compute instances."
```
Comment (xingao267) [inline templates/tfengine/recipes/org/monitor.hcl]: for Cloud SQL

Comment (xingao267) [reply templates/tfengine/recipes/org/monitor.hcl]: nvm. This is fixed in a new commit.

**Pull request merged**